### DISCUSSION TITLE
Sensor node production deployment on a VM

### DISCUSSION THREAD

**Discussion by rohan0017** [Security-Onion-Solutions/securityonion#11756] [General] [answered]
-I want to deploy a production sensor node on a VM. Will it work efficiently.  

-What are the minimum specs for production grade sensor mode deployment on a VM

**Comment by dougburks** [comment, marked as answer]
> -I want to deploy a production sensor node on a VM. Will it work efficiently.

From https://docs.securityonion.net/en/2.4/best-practices.html#best-practices:
_For production deployments, prefer dedicated hardware to VMs when possible (see the [Hardware Requirements](https://docs.securityonion.net/en/2.4/hardware.html#hardware) section)._

From https://docs.securityonion.net/en/2.4/hardware.html#virtualization:
_We recommend dedicated physical hardware (especially if you’re monitoring lots of traffic) to avoid competing for resources. Sensors can be virtualized, but you’ll have to ensure that they are allocated sufficient resources._

> -What are the minimum specs for production grade sensor mode deployment on a VM

https://docs.securityonion.net/en/2.4/hardware.html#cpu
https://docs.securityonion.net/en/2.4/hardware.html#ram
https://docs.securityonion.net/en/2.4/hardware.html#id1
